# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sane-backends/fujitsu.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We want to be able to scan A4 :-)

diff -urN sane-backends-1.0.19/backend/fujitsu.c sane-backends-1.0.19-fj/backend/fujitsu.c
--- sane-backends-1.0.19/backend/fujitsu.c	2008-01-23 02:21:21.000000000 +0100
+++ sane-backends-1.0.19-fj/backend/fujitsu.c	2008-02-20 12:04:46.000000000 +0100
@@ -1540,14 +1540,14 @@
     s->resolution_y = s->resolution_x;
   }
 
-  /* page width US-Letter */
+  /* max page width */
   s->page_width = 8.5 * 1200;
   if(s->page_width > s->max_x){
     s->page_width = s->max_x;
   }
 
-  /* page height US-Letter */
-  s->page_height = 11 * 1200;
+  /* max page height */
+  s->page_height = 14 * 1200;
   if(s->page_height > s->max_y){
     s->page_height = s->max_y;
   }
